BUILDING.md

```diff
@@ -2,7 +2,7 @@
 
 Depending on what platform or features you require, the build process may
 differ slightly. After you've successfully built a binary, running the
-test suite to validate that the binary works as intended is a good next step.
+test suite to validate that the binary works as intended are a good next step.
```

Comment:
I'm not sure I agree with this. Although the test suite is made up of many tests, the collection itself is what is being referred to here, which is singular (one test suite).